.github/workflows/s3-tests.yml

```diff
@@ -0,0 +1,318 @@
+          docker images
+          echo "=========================================="
+
+          echo "Check docker ps"
+          docker ps
+          echo "=========================================="
+
+          echo "Check neo-go version"
+          neo-go --version
+          echo "=========================================="
+
+          echo "Check neofs-s3-authmate version"
+          neofs-s3-authmate --version
+          echo "=========================================="
+
+          echo "Check neofs-s3-gw version"
+          echo "=========================================="
+          neofs-s3-gw --version
+          echo "=========================================="
+
+          echo "Check neofs-adm version"
+          neofs-adm --version
+          echo "=========================================="
+
+          echo "Check neofs-cli version"
+          neofs-cli --version
+          echo "=========================================="
+
+          echo "Check vendor dir"
+          ls -lah "${GITHUB_WORKSPACE}/neofs-dev-env/vendor"
+          echo "=========================================="
+
+        working-directory: neofs-dev-env
+
+################################################################
+
+      - name: Copy test-env file to .env for neofs-s3-dev-env
+        run: |
+          cp .env ${GITHUB_WORKSPACE}/neofs-s3-dev-env/.env
+        working-directory: neofs-dev-env
+
+      - name: Copy neofs-s3-authmate for neofs-s3-dev-env
+        run: |
+          cp ${GITHUB_WORKSPACE}/neofs-dev-env/vendor/neofs-s3-authmate bin/s3-authmate
+        working-directory: neofs-s3-dev-env
+
+      - name: Prepare test S3-gw test env
+        run: |
+          make prepare.s3-gw
+        working-directory: neofs-s3-dev-env
+
+      - name: Copy s3tests config to s3-tests directory
+        run: |
+          cp services/s3-gw/s3tests.conf ${GITHUB_WORKSPACE}/s3-tests/${S3_TESTS_CONFIG}
+        working-directory: neofs-s3-dev-env
+
+      - name: Prepare s3tests config
+        run: |
+          sed -i -e "s/host =.*$/host = ${S3_GW_IP}/" ${S3_TESTS_CONFIG}
+          sed -i -e "s/port =.*$/port = ${S3_GW_PORT}/" ${S3_TESTS_CONFIG}
+          sed -i -e "s/is_secure =.*$/is_secure = True/" ${S3_TESTS_CONFIG}
+          sed -i -e "s/ssl_verify =.*$/ssl_verify = False/" ${S3_TESTS_CONFIG}
+        working-directory: s3-tests
+
+      - name: Test config
+        run: |
+          cat ${S3_TESTS_CONFIG}
+        working-directory: s3-tests
+
+      - name: Setup test env
+        run: |
+          python3.8 -m venv virtualenv
+          ./bootstrap
+        working-directory: s3-tests
+
+      - name: Run Sanity tests for pull requests
+        timeout-minutes: 60
+        if: github.event_name == 'pull_request'
+        run: |
+          source virtualenv/bin/activate
+          S3TEST_CONF=${S3_TESTS_CONFIG} pytest --alluredir=${GITHUB_WORKSPACE}/allure-results -v -s $(tr '\n' ' ' <${GITHUB_WORKSPACE}/neofs-s3-gw/.github/sanity-tests)
```

Comment:
Maybe it would be better to mark them in the test code with the @pytest.mark.sanity decorator ?

Reply:
Fixed. marked tests with @attr('sanity')

Reply:
https://github.com/nspcc-dev/s3-tests/pull/36